presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestIntegrationSmokeTest.java

```diff
@@ -122,6 +122,45 @@ public void testDescribeTable()
         assertTrue(expectedColumnsPossibilities.contains(actualColumns), String.format("%s not in %s", actualColumns, expectedColumnsPossibilities));
     }
 
+    @Test
+    public void testInformationSchemaTablesWithoutEqualityConstraint()
+    {
+        String catalog = getSession().getCatalog().get();
+        assertQuery(
+                "SELECT table_name FROM information_schema.tables " +
+                        "WHERE table_catalog = '" + catalog + "' AND table_schema LIKE 'tpch' AND table_name LIKE '%orders'",
```

Comment:
We shouldn't need a catalog filter, since the `information_schema` is specific to a single catalog.

Reply:
Right. I wanted to retain this test as it originally was in `TestHiveIntegrationSmokeTest` (i simplified it a bit, but without modifying the semantics). I will add additional test case here.

Reply:
See new `Add more information_schema tests` commit